scrapers/scrape_zg.py

```diff
@@ -33,6 +33,17 @@
     data[row['Stand']][row['Typ']] = row['Anzahl']
 days = list(data.keys())
 last_day = data[days[-1]]
+
+csv_url = 'https://www.zg.ch/behoerden/gesundheitsdirektion/statistikfachstelle/daten/themen/result-themen-14-03-06-e1.csv'
+d_csv = sc.download(csv_url, silent=True)
+reader = csv.DictReader(StringIO(d_csv), delimiter=',')
+data2 = collections.defaultdict(dict)
```

Comment:
I'd prefer to only use one source, if the GitHub source is no longer updated, just replace it with the new CSV URL from the website, no need for `data` and `data2`.